pallets/parachain-staking/src/lib.rs

```diff
@@ -21,29 +21,28 @@
 //! This is different from `frame/pallet-staking` where you approval vote and then run Phragmen.
 //!
 //! ### Rules
-//! There is a new round every `BlocksPerRound` blocks.
+//! There is a new round every `<Round<T>>::get().length` blocks.
 //!
 //! At the start of every round,
 //! * issuance is distributed to collators for `BondDuration` rounds ago
 //! in proportion to the points they received in that round (for authoring blocks)
 //! * queued collator exits are executed
 //! * a new set of collators is chosen from the candidates
 //!
-//! To join the set of candidates, an account must call `join_candidates` with
-//! stake >= `MinCollatorCandidateStk` and fee <= `MaxFee`. The fee is taken off the top
-//! of any rewards for the collator before the remaining rewards are distributed
-//! in proportion to stake to all nominators (including the collator, who always
-//! self-nominates).
```

Comment:
I thought this detail was helpful, but I'm fine either way.

Reply:
True, the [official docs](https://docs.moonbeam.network/staking/overview/#reward-distribution) do a good job explaining this. I took it out because it's no longer a customizable commission per collator. Now it's uniform 20% collator commission.